### DISCUSSION TITLE
[Question] Download servers so slow!!

### DISCUSSION THREAD

**Discussion by zippy77777** [kubilus1/autoortho#445] [Q&A]
Windows 11
-XPlane 11
AO version 7

Does anyone know why this is? I mean 0.16MBps nearly all the time, it's impossible to download anything?

**Comment by zippy77777** [comment]
![image](https://github.com/kubilus1/autoortho/assets/33657120/bce498c5-0654-4305-8423-c65a8269fc81)
crazy!!

**Reply by zippy77777** [reply]
I am asking on so many peoples behalves...

**Reply by Jeff2122** [reply]
I think v0.7.1 released the other day is supposed to fix that slow download. What version are you using?

**Reply by DarkDomi5865** [reply]
Same for me. Mine is a bit better tho, 0.51 Mbps!

**Reply by zippy77777** [reply]
I am using v7. I looked and didn't see v7.1. I'll look again. Thx
